### ISSUE TITLE
Weekly reflections are sent during the baseline week

### ISSUE THREAD

**Issue by nickdotreid** [kpwhri/heartsteps#161]
By looking at the new participants in U01, I realized they have been sent a weekly survey even though they are not receiving any other interventions because they haven't completed the baseline period.

I don't think this was intended, so I'm logging the error here so we can prioritize it later.

**Comment by klasnja** [comment]
Yup, they should not be sent the weekly survey during the baseline week. (:


On June 15, 2020 at 9:50:05 AM, Nick Reid (notifications@github.com) wrote:

By looking at the new participants in U01, I realized they have been sent a
weekly survey even though they are not receiving any other interventions
because they haven't completed the baseline period.

I don't think this was intended, so I'm logging the error here so we can
prioritize it later.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/kpwhri/heartsteps/issues/161>, or unsubscribe

**Comment by jup014** [comment]
We need to look into this. Is it still happening?
